services/movie.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids).distinct()
+
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids).distinct()
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
```

Comment:
The return type for `create_movie` should be `Movie` instead of `QuerySet` since it returns a single instance of `Movie`, not a queryset.